### DISCUSSION TITLE
AppControl Manager: Some options such as 'Managed installer' missing; could this be added as an option for base policies?

### DISCUSSION THREAD

**Discussion by bladehstream** [HotCakeX/Harden-Windows-Security#437] [General]
As above, most enterprises have some kind of managed installer and it would be really useful to include that as an option in the base policy templates.

**Comment by bladehstream** [comment]
I can see it can be added in the 'configure policy rule options' section, it would be nice for it to be a 1-step process to creation. however, I AM lazy...

Please feel free to close if you want as this is a super low priority.

**Comment by HotCakeX** [comment]
I understand the main problem here which is having to browse for XML files in every page in the app, I'll fix it completely by implementing new functionalities and behaviors that reduce the need to do repetitive tasks.

They'll probably land in version 1.6 or 1.7.

**Comment by HotCakeX** [comment]
Hi, I implemented a Sidebar that lets you select a XML file path and reuse it in multiple parts of the app.

More info here: https://github.com/HotCakeX/Harden-Windows-Security/releases/tag/AppControlManager.v.1.7.0.0

Sidebar documentation: https://github.com/HotCakeX/Harden-Windows-Security/wiki/Sidebar

**Comment by HotCakeX** [comment]
Managed Installer policy creation is now available in AppControl Manager: https://github.com/HotCakeX/Harden-Windows-Security/releases/tag/AppControlManager-v2.0.728.0
